04-gas/gas-section-one/gas-section-one.html

```diff
@@ -28,8 +28,8 @@ <h1 class="gas-section-one-title">Lower your water temperature</h1>
       <div class="gas-section-one-img-container">
         <img
         class="gas-section-one-img"
-        src="../../00-overall/05-images/04-gas/gas-section-one.webp"
-        alt="image section one"
+        src="../../00-overall/05-images/04-gas/gas-section-one-boiler.png"
+        alt="the image of sofa with a picture behind that"
```

Comment:
this alt tag is incorrect.. the image says boiler.png

so it would be "an image of a boiler"